.env

```
@@ -1,2 +1,7 @@
-WEBSITE_URL = 'http://localhost:3000'
-LOCALHOST = 'localhost'
+REACT_APP_API_HOST=localhost
+REACT_APP_WEBSITE_URL=http://localhost:3000
+
+REACT_APP_FALLBACK_LANGUAGE=en
+REACT_APP_MAIN_LANGUAGE=uk
+
+REACT_APP_API_BASE_URL=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)